src/main/java/org/breedinginsight/services/validators/TraitValidatorService.java

```diff
@@ -166,6 +171,38 @@ public ValidationErrors checkTraitDataConsistency(List<Trait> traits, TraitValid
         return errors;
     }
 
+    public ValidationErrors checkTraitFieldsLength(List<Trait> traits, TraitValidatorErrorInterface traitValidatorErrors) {
```

Comment:
Doesn't look like this is actually called as part of the trait error checking in checkAllTraitValidations.

Reply:
Good catch, added in most recent commit.